05.ls/ls_1.rb

```diff
@@ -1,21 +1,35 @@
 # frozen_string_literal: true
 
 find_files = Dir.glob('*')
+max_size = find_files.map(&:size).max
 files = []
-find_files.each_slice(5) do |s|
+all_files = find_files.size
+n = if (all_files % 3).zero?
+      all_files / 3
+    else
+      all_files / 3 + 1
+    end
+find_files.each_slice(n) do |s|
   files << s
 end
```

Comment:
```suggestion
files = find_files.each_slice(n).to_a
```

のようにするとサクッと配列に変換できます

Reply:
以前教えて頂いていたのに元のコードに戻ってしまっていました🙇‍♂️